docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  db:
+    container_name: respa-db
+    build: ./docker/postgres/
+    restart: always
+    environment:
+        POSTGRES_USER: respa
+        POSTGRES_PASSWORD: respa
+        POSTGRES_DB: respa
+    ports:
+        - "5432:5432"
```

Comment:
This will share the database on publicly available port, which is a security issue. (Think cases when you're accessing a publicly available network with your development laptop.)

Anyway, is this really needed? It would still be possible to connect to the db server from the host via `docker exec` or by using the containers IP address (visible via `docker inspect`).

Reply:
Hmm.. I guess we could omit it. Will remove.